charts/consul/templates/crd-peeringacceptors.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if and .Values.connectInject.enabled .Values.global.peering.enabled }}
+{{- if .Values.controller.enabled }}
```

Comment:
Regenerating manifests also modified these. I'm assuming someone edited manually or didn't run the generation on a previous PR. Please advise.

Reply:
This should be reverted. Our regeneration is not accurate for the CRDs introduced via peering but we haven't had to change to update our generation scripts. This is true for the peering acceptor and peering dialer.